### ISSUE TITLE
Nextcloud desktop sync status indicator

### ISSUE THREAD

**Issue by AtomToast** [bilelmoussaoui/Hardcode-Tray#613]
### Specifications
- Hardcode-Tray version : `4.3`
- Distribution : `arch linux`
- Gtk version : `3.24`
- Cairo version : `2.4.2`
- Sni-qt version: `sni-qt-patched-git r.-1`
- Application name :  `nextcloud-client`
- Application version : `2.6.2`

### Expected behavior and actual behavior
I expect the icon to become monochrome but it stays green.  
I'm pretty sure I used this program before to change the icon to this (the circle with the checkmark) on my Manjaro KDE machine a long time ago:  
![](https://cdn.discordapp.com/attachments/454610005656862722/663753705316941835/unknown.png)  
However there appears to be no change on my arch dwm laptop:
![](https://cdn.discordapp.com/attachments/454610005656862722/663753425233772555/unknown.png)

### Steps to reproduce the problem
1. install nextcloud-client
2. apply harcode tray (I tried using CairoSVG, RSVGConvert, ImageMagick. Theme I am using is Papirus-Dark)
3. The fix doesn't apply to the application (only to discord and telegram for me)

**Comment by varlesh** [comment]
i think nextcloud use icons for tray from own themes in binary:
https://github.com/nextcloud/desktop/blob/master/src/gui/systray.cpp#L54
https://github.com/nextcloud/desktop/tree/master/theme

**Comment by AtomToast** [comment]
There is an option in the settings for this. Ignore this then w(
![image](https://user-images.githubusercontent.com/27757087/71963716-29208600-31f4-11ea-900f-f4a995f19fe7.png)

**Comment by renyhp** [comment]
I am in the same situation. In my previous install (Ubuntu 20.04) this icon was exactly like the screenshot in the first post. Now (Linux Mint 21.1) I can't make it themed anymore. The "Use Monochrome Icons" switch does not help, it just becomes monochrome but does not use the Papirus icon:

![image](https://user-images.githubusercontent.com/20142663/210231582-5c4cbcd9-77ec-4762-9837-9d3e6a5bd077.png)

I installed nextcloud-desktop from the official ppa (`3.6.4-20221208.195402.3ffb9827b-1.0~jammy1`), and papirus is also from the ppa (`20221201-6106+pkg22~ubuntu22.04.1`).

The hardcode-tray logs don't mention nextcloud.

EDIT: In fact, I am just realizing my screenshot shows that the blueman applet also isn't themed. Maybe I have some misconfiguration with my tray?

EDIT2: I found out that adding 
```
.panel {
 -gtk-icon-style: regular;
}
```

to my `.config/gtk-3.0/gtk.css` fixes the blueman icon. I'm a bit puzzled: I don't quite understand symbolic vs regular icons, I don't understand why the "Symbolic icons" checkbox in statusnotifier does nothing, and I don't understand if Papirus is supposed to provide the missing blueman symbolic icon. The Nextcloud icon is still unthemed, though...
